.github/workflows/lint.yaml

```diff
@@ -11,15 +11,25 @@ permissions:
   # pull-requests: read
 
 jobs:
-  golangci:
-    name: lint
+  lint:
+    name: golangci-lint
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
       - uses: actions/setup-go@v5
         with:
           go-version-file: 'go.mod'
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@v7
+        run: |
+          make lint
+  modernize:
+    name: modernize
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
         with:
-          version: v2.0.2
+          go-version-file: 'go.mod'
+      - name: modernize
+        run: |
+          make modernize
```

Comment:
We only need one of these now that the makefile update has gone through. Lets rename the `golangci-lint` to just `lint` and remove the `modernize` one

Reply:
Fixed.